.travis.yml

```diff
@@ -4,7 +4,7 @@ go:
   - 1.7
 
 install:
-  - ./install.sh
+  - go get -t ./...
```

Comment:
No, only with cosi, which 
1) should not be needed anymore
OR 
2) should be used with crypto as  @ineiti suggests